Examples/Scripts/Propagation/propagation_summary.py

```diff
@@ -0,0 +1,128 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import argparse
+import uproot
+import seaborn as sns
+
+p = argparse.ArgumentParser()
+
+p.add_argument(
+    "-i",
+    "--input",
+    nargs="+",
+    type=str,
+    default="",
+    help="Input file(s) with propagation summary",
+)
+
+p.add_argument(
+    "-l",
+    "--label",
+    nargs="+",
+    type=str,
+    default="",
+    help="Input label(s) for the propagation summary",
+)
+
+p.add_argument(
+    "-c",
+    "--color",
+    nargs="+",
+    type=str,
+    default="",
+    help="Input label(s) for the propagation summary",
+)
+
+p.add_argument(
+    "-m",
+    "--marker",
+    nargs="+",
+    type=str,
+    default=["o"],
+    choices=[
+        "o",
+        "s",
+        "D",
+        "v",
+        "^",
+        "<",
+        ">",
+        "p",
+        "P",
+        "*",
+        "h",
+        "H",
+        "+",
+        "x",
+        "X",
+        "d",
+    ],
+    help="Input label(s) for the propagation summary",
+)
+
+p.add_argument(
+    "-o",
+    "--output",
+    type=str,
+    default="",
+    help="Output file base name",
+)
+
+args = p.parse_args()
+
+try:
+
+    fig, ax = plt.subplots(1, 1, figsize=(11, 10))
+
+    eta_bins = np.linspace(-4, 4, 100)
+    eta_centers = 0.5 * (eta_bins[:-1] + eta_bins[1:])
+    eta_width = 8.0 / 100
+
+    for ir, irfile in enumerate(args.input):
```

Comment:
We could be a bit nicer with 
```suggestion
    for irfile, label, marker in zip(args.input, args.label, args.marker):
```